lib/routes/account.js

```diff
@@ -390,6 +395,26 @@ module.exports = function (
             )
         }
 
+        function checkNumberOfActiveSessions () {
+          return db.sessions(emailRecord.uid)
```

Comment:
Nah, possibly only a comment would help `// save results for later`, but I merged already :D